### PR TITLE
fix: search miner for eth2

### PR DESCRIPTION
**Pull request by m-r-g-t**
## What does this PR do?

A short description of what the PR does.

## Related issue

Link to the issue this PR addresses.

If there isn't already an open issue, create an issue first. This will be our home for discussing the problem itself.

## Testing

What testing was performed to verify this works? Unit tests are a big plus!

## Checklist before merging
- [x] Read the [contributing guide](https://github.com/flashbots/mev-inspect-py/blob/main/CONTRIBUTING.md)
- [x] Installed and ran pre-commit hooks
- [ ] All tests pass with `./mev test`

### CODE REVIEW

File: tests/blocks/10921990.json
Comment (wei3erHase) [inline tests/blocks/10921990.json:1]: uncertain about the naming of this file, is it really that block?
or can we name it like `1092...91-no-miners.json`?

File: tests/test_block.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    # instantiate the mock
+    mock_web3_instance = mocked_web3.return_value
+    mock_eth = mock_web3_instance.eth
+    mock_eth.get_block.return_value = {
+        "miner": "0x4a536c1f6a5d5a9c1aeca9f6d04fbbf5f0d8f4e3"
+    }
+
+    # load a sample block and remove the miner
+    block_number = 10921991
```
Comment (wei3erHase) [inline tests/test_block.py:28]: hmmm, why do we need the '90 file then?

File: tests/test_block.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+from unittest.mock import MagicMock, patch
```
Comment (wei3erHase) [inline tests/test_block.py:1]: i understand that this allows to static testing, is there a way of providing also prod tests like the rest?

**Review by wei3erHase — APPROVED** [review]
lgtm! 🚀

File: tests/test_block.py (hunk 2 above)
Comment (m-r-g-t) [reply tests/test_block.py:1]: afaik the others aren't prod tests - they are assuming the block was correctly decoded and operating on the json

**Pull request merged**